www/include/Administration/brokerPerformance/brokerPerformance.php

```diff
@@ -50,7 +50,11 @@ function createArrayStats($arryFromJson)
         $io[_('State')]['value'] = $arryFromJson['state'];
         if ($arryFromJson['state'] == "disconnected") {
             $io[_('State')]['class'] = "badge service_critical";
-        } elseif ($arryFromJson['state'] == "listening" || $arryFromJson['state'] == "connected" || $arryFromJson['state'] == "connecting") {
+        } elseif (
+            $arryFromJson['state'] == "listening"
```

Comment:
```suggestion
            $arrayFromJson['state'] == "listening"
```
could you change this variable name everywhere in the file ?